lib/ecto/query/planner.ex

```diff
@@ -1287,11 +1287,11 @@ defmodule Ecto.Query.Planner do
     end
   end
 
-  defp normalize_select(%{select: nil} = query, _keep_literals?) do
+  defp normalize_select(%{select: nil} = query, _keep_literals?, _allow_alias?) do
     {query, nil}
   end
 
-  defp normalize_select(query, keep_literals?) do
+  defp normalize_select(query, keep_literals?, allow_alias?) do
```

Comment:
I put the third parameter because it seemed cleaner to me than doing it outside `normalize_select`. But let me know what you think.